QUALITY_DECLARATION.md

```diff
@@ -138,21 +140,21 @@ It also has several test dependencies, which do not affect the resulting quality
 
 The `console_bridge_vendor` package provides a wrapper around `console_bridge`, providing nothing but a dependency on `console_bridge` on some systems. On others, it provides an [ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html) build of `console_bridge`.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
```

Comment:
These links should point to `foxy` branches

Reply:
Done @chapulina https://github.com/ros/class_loader/pull/174/commits/6c21f4f64b2987c7cb1611d9c536b9fa5f6a449b